cognite_toolkit/_cdf_tk/commands/deploy.py

```diff
@@ -353,15 +353,10 @@ def _create_resources(
         try:
             created = loader.create(resources)
         except CogniteAPIError as e:
-            if e.code == 409:
-                self.warn(LowSeverityWarning("Resource(s) already exist(s), skipping creation."))
-            else:
-                message = f"Failed to create resource(s). Error: {escape(str(e))!s}."
-                if hint := self._environment_variable_hint(
-                    loader.get_ids(resources), environment_variable_warning_by_id
-                ):
-                    message += hint
-                raise ResourceCreationError(message) from e
+            message = f"Failed to create resource(s). Error: {escape(str(e))!s}."
+            if hint := self._environment_variable_hint(loader.get_ids(resources), environment_variable_warning_by_id):
+                message += hint
+            raise ResourceCreationError(message) from e
```

Comment:
This `e.code == 409` is really bad as it hid the following error from the user:
![image](https://github.com/user-attachments/assets/5a097b3f-14e7-4926-99e3-cb8964d186cf)

Reply:
This error was caused by having a view implementing an edge view while having properties mapping to a node container. Thus the other fix.